File: src/main/java/io/github/skydynamic/increment/storage/lib/Interface/IDatabaseManager.java
package io.github.skydynamic.increment.storage.lib.Interface;

import java.util.UUID;

@SuppressWarnings("unused")
public interface IDatabaseManager {
    void setFileName(String name);

    void setDatabasePath(String path);

    void setCollectionUuid(UUID uuid);

    default String getFileName() {
        return "dataBase";
    }

    default String getDatabasePath() {
        return ".";
    }

    default UUID getCollectionUuid() {
        return UUID.randomUUID();
    }
}

File: src/main/kotlin/io/github/skydynamic/increment/storage/lib/manager/IDatabaseManager.kt
package io.github.skydynamic.increment.storage.lib.manager

import java.util.*

interface IDatabaseManager {
    fun setFileName(name: String?)

    fun setDatabasePath(path: String?)

    fun setCollectionUuid(uuid: UUID?)

    fun getFileName(): String {
        return "dataBase"
    }

    fun getDatabasePath(): String {
        return "."
    }

    fun getCollectionUuid(): UUID {
        return UUID.randomUUID()
    }
}